eng/versioning.targets

```diff
@@ -177,4 +177,10 @@
       WriteOnlyWhenDifferent="true" />
   </Target>
 
+  <Target Name="GenerateNativeSourcelinkFile"
+          DependsOnTargets="GenerateSourceLinkFile"
```

Comment:
Does this need to be incremental somehow? Eh inputs and outputs, or use AfterTargets which I believe piggy backs on the other target running

Reply:
AfterTargets will likely not work here - not all consumers of this are ready for this one. I can make the copy to only if newer so that it doesn't trigger downstream issues.